source/leuven/tier2_hardware/superdome_hardware.rst

```diff
@@ -1,19 +1,20 @@
 Superdome hardware
 ==================
-Superdome is a shared memory machine and consists of several nodes.
-
-
-.. include:: tier2_login_nodes.rst
-
+Superdome is a single node with a large amount of memory (6 TB)
+which is part of the Genius cluster. Please see the
+:ref:`Tier-2 access page<tier2_login_nodes>` for general information
+on how to access Genius.
 
 Hardware details
 ----------------
-- 8 nodes
+Superdome consists of
+
+- 8 sockets with
+
+    - Xeon Gold 6132 CPU\@2.6 GHz (Skylake), 14 cores each
+    - 750 GB RAM each
 
-    - Xeon Gold 6132 CPUs\@2.6 GHz (Skylake), 14 cores each
-    - 750 GB RAM
-    - partition ``superdome``
```

Comment:
- partition ``superdome`` and ``superdome_long``

Reply:
I don't think scheduler partition info should be copied to a hardware info page.
There is already a reference to the Superdome quickstart page where the Slurm partition info is provided.

Reply:
Well, one of the two partitions is already mentioned here, so, I do not see anything against mentioning the 2nd valid partition name. It is very relevant to this page too.

Unfortunately, in the Genius hardware page, we are only mentioning the 3-day partition names, and the 7-day partitions are only explained in the quick start page. The same for wICE. 
To me this is not completely OK, but this is a matter of taste.

What if a user comes up with a question like: "what are the possible/available partitions on Genius?". then on the VSCdocs, he does not find it all in the Genius hardware page; he only finds a partial answer. And without knowing about the other `_long` partitions, the user will stop searching, so, he might not figure it out.

I propose that we rename and slightly reformulate (of course in a separate PR) the Genius|wICE hardware pages, and call them like "Genius|wICE Hardware and Configuration" pages. Then, we can safely mention e.g. `gpu_p100_long` partition in that page, too.

But for now, the fact that `superdome_long` is not mentioned in the hardware page also complies with other Genius hardware page.

So, this PR is good to GO.